drone-webdriver/src/test/java/org/jboss/arquillian/drone/webdriver/binary/downloading/source/StubbedChromeStorageTestCase.java

```diff
@@ -70,4 +72,32 @@ public void should_find_latest_stable_release_with_32_bit_version() throws Excep
         assertThat(latestRelease.getVersion(), is(ALL_OS_RELEASE));
         assertThat(latestRelease.getUrl(), CoreMatchers.endsWith("win32.zip"));
     }
+
+    @Test
+    public void should_always_fallback_to_32_bit_version() throws Exception {
+        System.setProperty("os.name", "win");
+
+        assertThat(PlatformUtils.isWindows(), is(true));
+
+        final ExternalBinary releaseWithArchitectureDetected = chromeDrivers.getReleaseForVersion(ALL_OS_RELEASE);
+        assertThat(releaseWithArchitectureDetected.getVersion(), is(ALL_OS_RELEASE));
+        assertThat(releaseWithArchitectureDetected.getUrl(), CoreMatchers.endsWith("win32.zip"));
+
+        final ExternalBinary release32 = chromeDrivers.getReleaseForVersion(ALL_OS_RELEASE, Architecture.BIT32);
+        assertThat(release32.getVersion(), is(ALL_OS_RELEASE));
+        assertThat(release32.getUrl(), CoreMatchers.endsWith("win32.zip"));
+
+        final ExternalBinary release64 = chromeDrivers.getReleaseForVersion(ALL_OS_RELEASE, Architecture.BIT64);
+        assertThat(release64.getVersion(), is(ALL_OS_RELEASE));
+        assertThat(release64.getUrl(), CoreMatchers.endsWith("win32.zip"));
```

Comment:
Why not split it into 3 separate test cases?

Reply:
yes, I will split it

Reply:
see latest commit